pkg/apis/imageregistry/v1/types.go

```diff
@@ -90,7 +90,7 @@ type ImageRegistryConfigRoute struct {
 }
 
 type ImageRegistrySpec struct {
-	operatorsv1alpha1api.OperatorSpec `json:",inline"`
+	operatorsv1api.OperatorSpec `json:",inline"`
```

Comment:
what are we still using from this? can we get rid of it entirely?

Reply:
Checking...